docs/src/prototypes/meetingOptions/styles.ts

```diff
@@ -0,0 +1,13 @@
+import { CSSProperties } from 'react'
+
+export const middleColumnStyles = {
```

Comment:
will repeat one of the concerns point - as this is a clear indicator that we need to come up with better mechanism to integrate user's styles into theme (so that `siteVariables` and, maybe, `variables` will be consumed)

Reply:
Mention in the PR description